src/main/java/com/googlecode/jsonrpc4j/JsonRpcBasicServer.java

```diff
@@ -849,8 +848,12 @@ private JsonError writeAndFlushValueError(OutputStream output, ErrorObjectWithJs
 	 * @param value  the value to write
 	 * @throws IOException on error
 	 */
-	private void writeAndFlushValue(OutputStream output, Object value) throws IOException {
+	private void writeAndFlushValue(OutputStream output, ObjectNode value) throws IOException {
```

Comment:
Do you know for sure that this will always be of type `ObjectNode`?

Reply:
Yes. I have checked usages before change type. There are only two usages and it is ObjectNode in both cases.

Reply:
👍